Shopify/cli
# Use Shopify Functions Javy plugin instead of default Javy plugin

Do not merge this until a function-runner release including https://github.com/Shopify/function-runner/pull/397 has been released and this PR is updated to use that version of the function-runner.

### WHY are these changes introduced?

The Shopify Functions Javy plugin includes support for JavaScript APIs for Shopify Functions that are not included in the upstream default Javy plugin.

### WHAT is this pull request doing?

Switches the Javy plugin used while building JavaScript Shopify Functions from the default Javy plugin to the Shopify Functions Javy plugin.

### How to test your changes?

1. `pnpm clean`
2. `pnpm shopify app function build --path <path_to_js_function_extension>`
3. `wasm-tools print <path_to_js_function_extension>/dist/function.wasm | less` and check that the `canonical_abi_realloc` and `invoke` functions are being imported from `shopify_functions_javy_v1` instead of `javy_quickjs_provider_v3`.

### Measuring impact

How do we know this change was effective? Please choose one:


### Checklist
